### DISCUSSION TITLE
I don't find how to delete my account

### DISCUSSION THREAD

**Discussion by icanzilb** [TelemetryDeck/Viewer#205] [Q&A]
I'd like to delete my account completely (under EU regulation) but I don't find how to do that. What would you say the best way to do this is?

**Comment by winsmith** [comment]
Hi, there's no button, but I can do that for you of course. I'm sad to see you go though, do you have any feedback on how Lisa and I can do better?

**Reply by icanzilb** [reply]
Thank you please do. I think we already spoke a couple of times — I was only using td for my blog and td really doesn't provide out-of-the-box at least the basics that I need. I already migrated back to a tool that more suitable for a web site. Thanks & good luck

**Reply by winsmith** [reply]
Thank you very much! We're not ideal for websites and that's ok :) I hope you find the perfect tool for your needs. I'll delete your account, organization, and all associated data.
